modules/transport-netty4/src/main/java/org/elasticsearch/http/netty4/pipelining/HttpPipeliningHandler.java

```diff
@@ -120,4 +121,13 @@ public void write(final ChannelHandlerContext ctx, final Object msg, final Chann
         }
     }
 
+    @Override
+    public void close(ChannelHandlerContext ctx, ChannelPromise promise) throws Exception {
+        ClosedChannelException closedChannelException = new ClosedChannelException();
```

Comment:
Yeah that's what I mean by removing it completely from this class. Sorry for the confusion.